apps/wallet/src/features/Widget/CustomWalletLayout/FeatureButtons.tsx

```diff
@@ -0,0 +1,77 @@
+import type { FC } from 'react';
+import { useMemo } from 'react';
+import type { Networks } from '@walless/core';
+import { ArrowBottomRight, ArrowTopRight, Plus, Swap } from '@walless/icons';
+import WidgetButtons from 'components/WidgetButtons';
```

Comment:
if you rewrite a component for widgets, try to refactor and reuse it in the original features/screens